lab-doug/lib/storage.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
```

Comment:
Why did you use this one instead of the fs one? It's not a problem or anything, but the fs way is a little more sophisticated.